scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -1,3 +1,32 @@
 &.page-civicrm-a {
   @include civihr_table();
+
+  .crmCaseType {
+
+    .crm-button {
```

Comment:
shouldn't any button be styled automatically at this point? Why do the buttons on this screen require specific styling? Is the markup different?